src/platform/site-wide/announcements/config/index.js

```diff
@@ -22,6 +22,11 @@ const config = {
       paths: /disability-benefits\/apply\/$/,
       component: ClaimIncreaseBanner
     },
+    {
+      name: 'claim-increase',
+      paths: /^\/$/,
+      component: ClaimIncreaseBanner
+    },
```

Comment:
I think this takes the place of the `PersonalizationBanner`? We can probably remove that one from the list.

Reply:
:thinking: Isn't the path for the personalization banner is _any_ page (`/(.)/`), not just the home page? What happens if two banners are active at the same time?

Reply:
I believe it picks the first one. I had to put the claim increase banner above personalization.

Reply:
Oh whoops, misread your regex. That makes sense!